src/Messenger.js

```diff
@@ -117,8 +117,10 @@ class Messenger extends EventEmitter {
               throw new Error('Attachment size exceeds allowable limit.');
             case 2018001:
               throw new Error('No matching user found.');
+            case 2018034:
+              throw new Error('Message cannot be empty.');
             default:
-              throw new Error('Invalid fbid.');
```

Comment:
Invalid fbid isn't always the case here. Might be best to add an explicit check for when the subcode isn't set (in which case it definitely is the fbid), but for now I've just printed the error message in order to help devs determine what error has occurred.